internal/app/referenceclient/wire_details_test.go

```diff
@@ -609,9 +609,11 @@ func TestExamineGRPCEndStream(t *testing.T) {
 			} else {
 				// When there's feedback, the connect-go client may complain about the end-stream message
 				// and report a different code.
-				assert.True(t, connect.CodeOf(err) == connect.CodeAlreadyExists ||
-					connect.CodeOf(err) == connect.CodeInternal,
-					"unexpected error: %v", err)
+				assert.Contains(t,
+					[]connect.Code{connect.CodeAlreadyExists, connect.CodeInternal, connect.CodeUnknown},
```

Comment:
With fixes in latest connect-go, some of these scenarios can now result in "unknown" code.